### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by andresporras3423**
We have created the video clone of the youtube page using bootstrap and visual code. The navbar, video section, video info and the aside section with the miniature of the recommended videos are all added to the page.

### CODE REVIEW

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### :clap: Hi @andresporras3423 , @sergiomauz .
> **Your project looks nice but Bootstrap and jquery are beyond the scope of this project based on the curriculum.**

> **Kindly remake your page using plain HTML and CSS and submit another code review request**

### Status: Changes Requested.

**Comment by andresporras3423** [comment]
Not using bootstrap anymore. Now working with flexbox and CSS grid.

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
# Status [APPROVED]
## Great job on working on all requirements :+1: 


Go ahead and merge to master.
Good luck in your next project Happy coding

File: style.css [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+
+@media (max-width: 991px) {
+  .aside-styles {
+    background-color: white;
+    padding: 5px;
+    margin-top: 15px;
+  }
+
+  .no-border:active {
+  }
```
Comment (Anna-Myzukina) [inline style.css]: Line number 343 Do not use empty rulesets, better to delete

Comment (sergiomauz) [reply style.css]: Done!. Thank you Anna.

**Pull request merged**